plugins/connection/aws_ssm.py

```diff
@@ -433,6 +436,17 @@ def filter_ansi(line: str, is_windows: bool) -> str:
     return line
 
 
+@dataclass
```

Comment:
I would have used a typed dict instead of a dataclass, but it's ok.

Reply:
We've seen both starting to appear, is this something we want to be opinionated about?